WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/AddOrderCoordinator.swift

```diff
@@ -86,3 +107,34 @@ private extension AddOrderCoordinator {
         ServiceLocator.analytics.track(event: WooAnalyticsEvent.Orders.orderAddNew())
     }
 }
+
+final class NewSimplePaymentsNoticeViewController: UIViewController {
+
```

Comment:
We don't need this extra empty line here, same below

Reply:
Cleaned up here 5ab165eb82d811157aaf9f236ed898662eb82fd1